scipy/spatial/ckdtree.pyx

```diff
@@ -451,13 +451,8 @@ cdef public class cKDTree [object ckdtree, type ckdtree_type]:
         Apply a m-d toroidal topology to the KDTree.. The topology is generated 
         by :math:`x_i + n_i L_i` where :math:`n_i` are integers and :math:`L_i`
         is the boxsize along i-th dimension. The input data shall be wrapped 
-        into :math:`[0, L_i)`. A ValueError is raised if any of the data is
+        into :math:`[0, L_i)`. A ValueError is raised if any of the data is 
```

Comment:
you may want to set up your code editor so it automatically strips trailing whitespace